docs_user/modules/proc_stopping-infrastructure-management-and-compute-services.adoc

```diff
@@ -38,7 +38,7 @@ endif::[]
 
 .Procedure
 
-* Remove the conflicting repositories and packages from all Compute hosts:
+* Stop the Pacemaker services:
```

Comment:
Also see line 9 above, there is another mention of the "remove conflicting repositories" which should be removed.